Automattic/wp-calypso
# Virtual themes: Fix site preview for default variation

## Proposed Changes

Fixes an issue that was preventing the default style variation from being previewed after selecting a virtual theme

## Testing Instructions

- In `wordpress.com` (production) go to `/setup/site-setup/designSetup?siteSlug=<SITE_DOMAIN>&flags=virtual-themes/onboarding`
- Select a virtual theme to open the "site preview"
- While in the "site preview", select the default style variation
- Note how it is not previewed
- Now use the Calypso live link below and repeat the same steps
- Make sure the default style variation is previewed
